examples/calib_pulses.py

```diff
@@ -0,0 +1,91 @@
+r"""
+Calibrate pulses output from RF amplifier
+=========================================
+A series of betas ($\mu$s $\sqrt{W}$) are supplied to the spincore to output to
+the RF amplifier. The output of the amplifier is then captured on the GDS
+oscilloscope. It is very important to note that there MUST BE at least a 40 dBm
+attenuator between the RF amplifier output and the GDS oscilloscope input to
+avoid damaging the instrumentation! It is advised that the attenuator be
+calibrated using the GDS and AFG beforehand
+"""
+import pyspecdata as psd
+import os
+import SpinCore_pp as spc
+from datetime import datetime
+from Instruments import GDS_scope
+import numpy as np
+
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+# {{{ importing acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+# {{{ add file saving parameters to config dict
+config_dict["type"] = "pulse_calib"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["misc_counter"] += 1
+# }}}
+# {{{ ppg
+sqrt_P = config_dict["amplitude"] * np.sqrt(75)  # we have a 75 W amplifier
+desired_beta = np.linspace(0.5*sqrt_P, 100, 50)
+tx_phases = np.r_[0.0, 90.0, 180.0, 270.0]
+Rx_scans = 1
+datalist = []
+prog_t = []
+# {{{ set up settings for GDS
+with GDS_scope() as gds:
+    gds.reset()
+    gds.CH2.disp = True
+    gds.write(":CHAN1:DISP OFF")
+    gds.write(":CHAN2:DISP ON")
+    gds.write(":CHAN3:DISP OFF")
+    gds.write(":CHAN4:DISP OFF")
+    gds.CH2.voltscal = 100e-3  # set voltscale to 100 mV
+    gds.timescal(50e-6, pos=0)  # set timescale to 50 us
+    gds.write(":TIM:MOD WIND")
```

Comment:
What does this do? Add a comment?

Reply:
This was pulled from capture_gds.py (used in generating test signal) but after reading docs we don't want this. Good catch!

Reply:
it sets the time based mode to zoom the window. We don't want to do this since I'm pretty sure it would negate the setting of the voltscale

Reply:
no, but I don't think we need it.